src/controllers/boolController.js

```diff
@@ -0,0 +1,26 @@
+const { negate, truthiness, isOdd, startsWith } = require('../lib/booleans');
+
+exports.bool_negate = (req, res) => {
+  res.status(200).json({ result: negate(req.body.value) });
+};
+
+exports.bool_truthiness = (req, res) => {
+  res.status(200).json({ result: truthiness(req.body.value) });
```

Comment:
Here we are not returning, but below you are using a return before the ternary logic, it's good practice to be consistent with returns and only return where you need to. See if you can remove your return for the ternaries? 

You can also use return to end your function if you had lots of logic and complicated response status and jsons